pkg/acl/access_control_list_middleware.go

```diff
@@ -41,6 +41,14 @@ func (middleware *AccessControlListMiddleware) Authorize(next http.Handler) http
 
 		orgId, _ := claims.GetOrgId()
 
+		if middleware.accessControlListConfig.EnableAccessList {
+			orgIsAccepted := middleware.accessControlListConfig.AccessList.IsUserAccepted(orgId)
+			if !orgIsAccepted {
+				shared.HandleError(r, w, errors.New(errors.ErrorForbidden, "organisation '%s' is not authorized to access the service during the current SSO Migration.", orgId))
```

Comment:
@akoserwal this is the temporary error message which we will replace :)

`"organisation '%s' is not authorized to access the service during the current SSO Migration."`

Reply:
I think a new error code will be required here. 

@Rajagopalan-Ranganathan @wtrocki @davidffrench What should the error message in the response during the migration?

Reply:
This message is too specific to end user.
I will recommend to create new error code. Once that is there I can take it from there with SDK and UI and CLI migration.
Messages are ignored by UI/CLI are are useful only for API users.
 
I would put error code as generic - `Service is under maintenance`

Reply:
We need issues for UI and CLI to handle that code so we would not forget about it/things land in backlog

Reply:
Thanks @akoserwal  and @wtrocki

Reply:
Hey @akoserwal & @wtrocki 
How do you feel about the error code that's been added? Let me know of any adjustments that should be made :)

Reply:
@VanillaSpoon it looks good to me